mitosheet/src/mito/components/endo/celleditor/CellEditor.tsx

```diff
@@ -485,10 +494,11 @@ const CellEditor = (props: {
             return;
         }
 
-        const columnID = props.sheetData.data[props.editorState.columnIndex].columnID;
-        const columnHeader = props.sheetData.data[props.editorState.columnIndex].columnHeader;
-        const formula = getFullFormula(props.editorState.formula, props.editorState.pendingSelections, props.sheetData)
-        const formulaLabel = props.sheetData.index[props.editorState.rowIndex];
+        const editorSheetData = props.sheetDataArray[props.editorState.sheetIndex];
+        const columnID = editorSheetData.data[props.editorState.columnIndex].columnID;
+        const columnHeader = editorSheetData.data[props.editorState.columnIndex].columnHeader;
+        const formula = getFullFormula(props.editorState, sheetData)
```

Comment:
Here's a great location where I'd rather not have to think about -- is this sheetData of selected sheet or sheet data of editor sheet?